content/packages-and-modules/contributing-packages-to-the-registry/specifying-dependencies-and-devdependencies-in-a-package-json-file.mdx

```diff
@@ -26,6 +26,10 @@ To add an entry to the `"devDependencies"` attribute of a `package.json` file, o
 
 ```
 npm install <package-name> --save-dev
+
+OR
+
+npm install -D <package-name> 
```

Comment:
```suggestion
npm install <package-name> -D
```
edit for consistency

Reply:
Thanks Gar it's my first time leaving feedback I hope I'm doing it right!